skills/cypilot/SKILL.md

```
@@ -121,6 +123,39 @@ ALWAYS use the resolved absolute path WHEN encountering a `{variable}` reference
+
+**Step 1 — Detect `cpt`**:
+```bash
+command -v cpt
+```
+
+**Step 2 — Set `{cpt_cmd}`**:
+- If `cpt` is found: `{cpt_cmd}` = `cpt`, `{cpt_installed}` = `true`
+- If `cpt` is NOT found: `{cpt_cmd}` = `python3 {cypilot_path}/.core/skills/cypilot/scripts/cypilot.py`, `{cpt_installed}` = `false`
+
+**Step 3 — If `cpt` is NOT found**, inform the user:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)